ui/app/src/modules/Preview/Tools/AssetsPanel.tsx

```diff
@@ -15,24 +15,232 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-import React from 'react';
-import Typography from '@material-ui/core/Typography';
-import { defineMessages } from 'react-intl';
+import React, { useEffect, useMemo } from 'react';
+import { defineMessages, useIntl } from 'react-intl';
 import ToolPanel from './ToolPanel';
+import { useEntitySelectionResource } from "../../../utils/hooks";
+import { ElasticParams, MediaItem } from "../../../models/Search";
+import { setRequestForgeryToken } from "../../../utils/auth";
+import { createStyles } from "@material-ui/core";
+import makeStyles from "@material-ui/core/styles/makeStyles";
+import SearchBar from '../../../components/SearchBar';
+import { useDispatch, useSelector } from "react-redux";
+import GlobalState, { PagedEntityState } from "../../../models/GlobalState";
+import TablePagination from "@material-ui/core/TablePagination";
+import { Subject } from "rxjs";
+import LoadingState from "../../../components/SystemStatus/LoadingState";
+import { debounceTime, distinctUntilChanged } from "rxjs/operators";
+import { DRAWER_WIDTH, getHostToGuestBus } from "../previewContext";
+import { ASSET_DRAG_ENDED, ASSET_DRAG_STARTED, fetchPanelAssetsItems } from "../../../state/actions/preview";
+import { ErrorBoundary } from "../../../components/ErrorBoundary";
+import MediaCard from '../../../components/MediaCard';
+import DragIndicatorRounded from '@material-ui/icons/DragIndicatorRounded';
+import EmptyState from "../../../components/SystemStatus/EmptyState";
 
 const translations = defineMessages({
   assetsPanel: {
     id: 'craftercms.ice.assets.title',
     defaultMessage: 'Assets'
+  },
+  itemsPerPage: {
+    id: 'search.itemsPerPage',
```

Comment:
Is this the exact same text and id from the search page?

Make sure they are the same or use a unique id for the Panel